app/components/UserValidator/index.tsx

```diff
@@ -76,16 +79,19 @@ const Validator = ({ handleSelect, validateAbakusGroup }: Props) => {
    * @param {string} result The newest scan
    * @returns void
    */
-  const addScanResult = (result: string) => {
+  const addScanResult = (result: ScanResult) => {
     setScanResults((prevScanResults) => {
       if (
         prevScanResults.length !== 0 &&
-        prevScanResults[0].message === result
+        prevScanResults[0].message === result.message
       ) {
-        prevScanResults[0].count++;
+        prevScanResults[0].count = (prevScanResults[0].count ?? 0) + 1;
```

Comment:
I feel like this should still be ?? 1, makes more sense

Reply:
oh okay